src/views/communities/index.tsx

```diff
@@ -119,32 +119,32 @@ function CommunitiesScreen() {
             if (communtiesOrder === 'nearest' && userLocation) {
                 Api.community
                     .list({
-                        offset: communtiesOffset + 5,
-                        limit: 5,
+                        offset: communtiesOffset + 10,
```

Comment:
why change to 10?

Reply:
This was an attempt to make less API calls in order to avoid the error `Warning: Encountered two children with the same key`, but there's a tradeoff since the calls are "heavier" than before. I changed back to 5 since the error still happens.